Resources/public/js/cookie_consent.js

```diff
@@ -54,10 +54,10 @@ document.addEventListener("DOMContentLoaded", function() {
 
     if (cookieConsentCategoryDetails && cookieConsentCategoryDetailsToggle) {
         cookieConsentCategoryDetailsToggle.addEventListener('click', function() {
-            var detailsIsHidden = cookieConsentCategoryDetails.style.display !== 'block';
-            cookieConsentCategoryDetails.style.display = detailsIsHidden ? 'block' : 'none';
-            cookieConsentCategoryDetailsToggle.querySelector('.ch-cookie-consent__toggle-details-hide').style.display = detailsIsHidden ? 'block' : 'none';
-            cookieConsentCategoryDetailsToggle.querySelector('.ch-cookie-consent__toggle-details-show').style.display = detailsIsHidden ? 'none' : 'block';
+            var detailsIsHidden = cookieConsentCategoryDetails.offsetParent !== null;
```

Comment:
I'm fine with this change, but then I'd change this to 
```suggestion
            var detailsIsHidden = cookieConsentCategoryDetails.offsetParent === null;
```

and revert the other lines